scripts/parallel-test.sh

```diff
@@ -1,35 +1,20 @@
 #!/bin/bash
-set -e # exit on failure of any "simple" command (excludes &&, ||, or | chains)
-# prepare environment
-PATH=$PATH:~/opt/mongodb/bin
-echo "Extracting build directory..."
-[[ -z "${1}" ]] && tar -zxf build.tar.gz || tar -xzf $1
-cd /data/job/build
-# run tests
-echo "Running tests..."
-CPU_CORES=$(getconf _NPROCESSORS_ONLN)
-echo "$CPU_CORES cpu cores detected."
-TEST_COUNT=$(ctest -N -LE _tests | grep -i 'Total Tests: ' | cut -d ':' -f 2 | awk '{print $1}')
-[[ $TEST_COUNT > 0 ]] && echo "$TEST_COUNT tests found." || (echo "ERROR: No tests registered with ctest! Exiting..." && exit 1)
+set -eo pipefail
+[[ -f build.tar.gz ]] && tar -xzf build.tar.gz
+cd ./build
+echo "[Running tests]"
 set +e # defer ctest error handling to end
-echo "$ ctest -j $CPU_CORES -LE _tests --output-on-failure -T Test"
```

Comment:
I am not opposed to removing this echo statement for the reasons Matt brought up in the past, but the purpose of it is so the engineers can see (in the logs) the exact command Buildkite is running to replicate it on their own machines.